homeassistant/components/teslemetry/coordinator.py

```diff
@@ -92,6 +92,7 @@ def __init__(self, hass: HomeAssistant, api: EnergySpecific) -> None:
             update_interval=ENERGY_LIVE_INTERVAL,
         )
         self.api = api
+        self.updated_once = False
```

Comment:
These are not used by the binary platform (Energy site)

Reply:
It was throwing a mypy error that they didnt exist because all these classes share a single parent at one point. Is it better to exclude that type error instead?